### PR TITLE
Brijesh CI FAQ

### PR DESCRIPTION
**Pull request by brijeshjagani**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by dewan-ahmed — APPROVED** [review]
LGTM from CI. Will take 👍🏽 from PL.

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 1]
```diff
@@ -1571,6 +1571,25 @@ In this example, the `actions` (pull, push) and the repository name are correctl
...
+
+For more details on running Docker-in-Docker in a CI stage, refer to [our documentation](https://developer.harness.io/docs/continuous-integration/use-ci/manage-dependencies/run-docker-in-docker-in-a-ci-stage/).
+
+### Does the GitHub connector support deleting a branch?
+While Harness doesn't have a direct feature to delete a branch, you can achieve this by using a shell script if you have the necessary permissions set up in GitHub.
+
+### How do we make a PR to a specific branch after pipeline changes in BitBucket?
+After editing your pipeline, click the "Save Pipeline" button. You should see a "Start a pull request to merge" checkbox. Use this to select the branch and raise a PR for your changes.
+
+### How can we update the build status display name in Bitbucket to be more descriptive?
```
Comment (nofarblue) [inline kb/continuous-integration/continuous-integration-faqs.md:1590]: suggest creating a run step to execute API command, to update the status as they see fit

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 2]
```diff
@@ -1571,6 +1571,25 @@ In this example, the `actions` (pull, push) and the repository name are correctl
 
 To avoid authentication issues, it's recommended to either use a PAT when configuring build and push steps for Docker registries with the V2 API or, if using a username and password, switch to the V1 API.
 
+### How can I build and upload linux/amd64 and linux/arm64 images simultaneously to ECR using Harness?
+Yes, you can build multi-platform images using Harness CI/CD. The Build and Push to ECR plugin supports building multi-architecture images. However, when using the built-in build and push steps, you'll need to execute separate stages for each architecture.
```
Comment (nofarblue) [inline kb/continuous-integration/continuous-integration-faqs.md:1575]: I think we shouldn't say it's supported with build and push - when customers are asking for multi-architecture they want to do it in one step execution.

**Review by nofarblue — CHANGES_REQUESTED** [review]

**Comment by dewan-ahmed** [comment]
@brijeshjagani pls address the MC under `kb/continuous-integration/continuous-integration-faqs.md` and Nofar's comments.

**Pull request closed without merging**